nan.h

```diff
@@ -1654,12 +1661,12 @@ class Callback {
       Execute(progress);
   }
 
-  void SendProgress_(const char *data, size_t size) {
-    char *new_data = new char[size];
+  void SendProgress_(T *data, size_t size) {
+    T *new_data = new T[size];
     memcpy(new_data, data, size);
```

Comment:
Well, it is guaranteed to work for anything that satisfies `std::is_trivially_copy_constructable`, but that trait is not available in C++03. I do agree that this cannot stand as-is.